index.d.ts

```diff
@@ -532,6 +536,13 @@ declare module 'plaid' {
     getCategories(cb: Callback<CategoriesResponse>): void;
 
     resetLogin: AccessTokenFn<ResetLoginResponse>;
+
+    // sandboxCreatePublicToken(String, Array<sring>,Object?, Function)
+    sandboxPublicTokenCreate(institutionId: string,
+                    initialProducts: Array<string>,
+                    options: Object,
```

Comment:
nit: should we provide a typing for the `options`? we seem to be somewhat inconsistent about this... so maybe its "out of scope".